apps/dsp-app/src/app/main/guard/ontology-class-instance.guard.ts

```diff
@@ -0,0 +1,49 @@
+import { ProjectService } from '@dasch-swiss/vre/shared/app-helper-services';
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, Router } from '@angular/router';
+import { Observable, combineLatest } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { AuthService } from '@dasch-swiss/vre/shared/app-session';
+import { RouteConstants } from '@dasch-swiss/vre/shared/app-config';
+import { Select } from '@ngxs/store';
+import { UserSelectors } from '@dasch-swiss/vre/shared/app-state';
+import { StoredProject } from '@dasch-swiss/dsp-js';
+
+
+@Injectable({
+    providedIn: 'root'
+})
+export class AddOntologyClassInstanceGuard implements CanActivate {
```

Comment:
Rename file from ontology-class-instance.guard.ts to add-ontology-class-instance.guard.ts or the opposite with the class